md/release-notes.md

```diff
@@ -42,6 +42,12 @@ To be documented
 
 ### API removals
 
+#### BDM Api removal
+Since 7.0, BDM APIs and model classes in packages `com.bonitasoft.engine.bdm.*` where deprecated in favor of the Community version in packages `org.bonitasoft.engine.bdm.*`.  
```

Comment:
"in _the `com.bonitasoft.engine.bdm.*`_ packages"

Reply:
not sure whether both formulations are accepted...